### DISCUSSION TITLE
It seems like the vue examples and the vue template have been removed, is this temporary?

### DISCUSSION THREAD

**Discussion by 8bitreed** [alephjs/aleph.js#569] [General] [answered]
Following the docs I've run `deno run -A -r https://alephjs.org/init.ts --template=vue`

But it responds:` Invalid template name vue, must be one of [react,react-mdx,yew,leptos,api]`

Is this just temporary, or are there no longer plans to support Vue.js?

**Comment by LePichu** [comment, marked as answer]
Heya, unfortunately at the moment, Aleph.js is entirely unmaintained, and there are no clear plans going forward with Aleph. X. (or jexia), the creator of Aleph, also makes and maintains [esm.sh](https://esm.sh) and most of their time goes into working on that. As for any alternatives, Lume is the only good one with a bit of setup for SSR, unless you want Vue, in which case nothing really supports it. The Ultra.js people are looking into Vue at the moment, so that's worth mentioning, you might wanna drop by Ultra Discord and ping @virtue_signal for any further answers.
